src/cmd/singularity/cli/cli_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"testing"
+
+	"github.com/magiconair/properties/assert"
+	"github.com/spf13/cobra"
+)
+
+var cmd cobra.Command
+
+func TestEnvAppend(t *testing.T) {
```

Comment:
if no privs needed to run this test, we should do
```go
	test.DropPrivilege(t)
	defer test.ResetPrivilege(t)
```

Reply:
I think contrarily, if the test doesn't specifically need privilege, I think it's OK to skip this